cdc/owner.go

```diff
@@ -1019,6 +1028,36 @@ loop:
 	return err
 }
 
+// watchCampaignKey watches the aliveness of campaign owner key in etcd
+func (o *Owner) watchCampaignKey(ctx context.Context) error {
+	key := fmt.Sprintf("%s/%x", kv.CaptureOwnerKey, o.session.Lease())
+	resp, err := o.etcdClient.Client.Get(ctx, key)
+	if err != nil {
+		return cerror.WrapError(cerror.ErrPDEtcdAPIError, err)
+	}
+	if resp.Count == 0 {
+		return cerror.ErrOwnerCampaignKeyDeleted.GenWithStackByArgs()
+	}
+restart:
+	wch := o.etcdClient.Client.Watch(ctx, key, clientv3.WithRev(resp.Header.Revision+1))
```

Comment:
Why `+1`?

Reply:
The change with revision `resp.Header.Revision` is not needed, so `+1`

Reply:
Could you add some comments?